File: test/bench/c/adds.c
int fibo(int n){
  int result;
	int a = 0;
	int b = 1;
	int i = 1;
  while( i < n )
  {
      result = a + b;
      a = b;
      b = result;
      i++;
  }
	return result;
}

int main(void)
{
  int counter = 50000;
	int fib ;
	while(counter) {
		fib = fibo(40);
    counter -= 1;
	}
}

File: test/bench/ruby/adds.rb
def fibo( n)
	 a = 0
	 b = 1
	 i = 1
  while( i < n ) do
    result = a + b
    a = b
    b = result
    i+= 1
  end
	return result
end

 counter = 50000

while(counter > 0) do
	fibo(40)
  counter -= 1
end
